### PR TITLE
Create navigation, footer and contact section

### PR DESCRIPTION
**Pull request by nicupop729**
### In this feature branch, the following modifications were implemented:

- create and style navigation
- create and style footer
- create and style contact
- add JavaScript behaviour on the navigation so that any navigation button is clicked, the corresponding section renders
- use Luxon API to display time on the page

### CODE REVIEW

**Comment by Code-panther** [comment]
This webhint error `AnalyzerError: EACCES: permission denied, scandir '/lost+found'` occurs only on Github. We did some research and found that it is a Linux permission issue. @nicupop729 uses a Linux system as well and has the same linter error locally. Meanwhile, I use a windows machine and the linters all pass successfully locally before pushing the remote repo.

What could be done about this nasty error?

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Nicu & Promise, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦  🚦  

## Good Points 👍  
- Descriptive Pull Request
- Correctly used the Luxon library
- Design mostly matches the wireframe
- App is running as intended
- Nice UI design


## Required Changes ♻️
_Kindly check the comments under the review._


<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -10,15 +10,31 @@
...
+          <ul class="nav-list">
+            <li class="nav-item">
+              <a class="nav-link" id="triggerBookDisplay" href="#">List</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" id="triggerBookForm" href="#">Add new</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" id="triggerContact" href="#">Contact</a>
+            </li>
```
[the remark this answers is not in the document]
Comment (nicupop729) [reply index.html]: Thanks a lot for the feedback! We fixed this issue!

**Review by barackm — APPROVED** [review]
Hi @nicupop729 and @Code-panther, 👋 

# STATUS: Approved  🎉 
Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**